BLSQ/iaso
# Polio-1187 lqas country list view

Add a list view tio LQAs so districts without shapes can be seen

Related JIRA tickets : POLIO-1187

## Self proofreading checklist

- [X] Are my typescript files well typed - Well kinda

## Changes

- Split the `LqasImMap` and added a specific `LqasCountryMap` to avoid breaking stuff in IM
- Add `LqasCountryListOverview` --> That's the table!
- Move some utils, constants and components  out of the `shared`folder` and grouped them with LQAS. Sorry, that means a lot of imports updated
- Add `LqasOverviewContainer` to handle the code and components shared by the MAp and list views (eg: round selector).
- Moved the API calls in `LqasOverviewContainer` because both district and region are needed for the list view. regions are used for districts without data: the payload from the datastore won't include the parent name if there's no lqas data.
- Changed link from Afro Map to country view so that the country view will show the data for the selected round: map on the left, list on the right

## How to test

With local datastores, just play around with LQAS country view and Afro Map

Also test IM to see if nothing broke

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/440963b9-e635-419a-9347-75924854e390


https://github.com/BLSQ/iaso/assets/38907762/7cca271a-0538-4390-8714-e42776e90a58


## Notes

merges into https://github.com/BLSQ/iaso/pull/821.

Parent PR should be merged first as changes are already part of this branch
